tools/run_tests/artifacts/build_package_python.sh

```diff
@@ -21,14 +21,33 @@ mkdir -p artifacts/
 
 # All the python packages have been built in the artifact phase already
 # and we only collect them here to deliver them to the distribtest phase.
-find "${EXTERNAL_GIT_ROOT}"/input_artifacts/ \
-    -maxdepth 1 \
-    -type d \
-    -name "${ARTIFACT_PREFIX}*" \
-    -not -name "${EXCLUDE_PATTERN}" \
-    -print0 \
-        | xargs -0 -I% find % -type f -maxdepth 1 -exec cp -v {} ./artifacts \;
-
-# TODO: all the artifact builder configurations generate a grpcio-VERSION.tar.gz
-# source distribution package, and only one of them will end up
-# in the artifacts/ directory. They should be all equivalent though.
+
+# Build the find command to include all files which start with ARTIFACT_PREFIX but does not match any of the EXCLUDE_PATTERNS
```

Comment:
Let's discuss this IRL. I'm ok if this works, but at the first glance it looks more complicated than it could've been. Ideally I'd like something simpler, more straightforward. And avoid `-f` when copying.